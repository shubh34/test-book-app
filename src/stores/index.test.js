import configureStore from './index';
describe('configureStore', () => {
  it('should create store', () => {
    const store = configureStore();
    expect(store).toMatchSnapshot();
    expect(store.getState()).toMatchSnapshot();
  });
});
