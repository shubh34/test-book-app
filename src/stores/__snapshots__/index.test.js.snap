// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configureStore should create store 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;

exports[`configureStore should create store 2`] = `
Object {
  "applicationConfig": Object {
    "books": Array [],
    "hasError": false,
    "isApiCallInProgress": Array [],
  },
  "books": Object {
    "books": Array [],
    "error": undefined,
    "hasError": false,
    "isApiCallInProgress": false,
  },
  "router": Object {
    "action": "POP",
    "location": Object {
      "hash": "",
      "key": "default",
      "pathname": "/",
      "query": Object {},
      "search": "",
      "state": null,
    },
  },
}
`;
